operations/iscsi_sbps/iscsi_workers.md

```diff
@@ -0,0 +1,68 @@
+# Operational procedure for changing iSCSI worker nodes
+
+In CSM 1.6.0, all worker nodes were configured as iSCSI SBPS targets by default when CSM is installed/upgraded.
+In CSM 1.7.0, selective node personalization is supported where user/admin can limit/configure the required worker
+nodes as iSCSI SBPS targets. This is done by creating HSM group named `iscsi_worker` and add worker node xnames to
+this group which are required to be configured as iSCSI targets. See ‘Steps to create HSM group’ section of [HSM groups](https://github.com/Cray-HPE/docs-csm/blob/release/1.7/operations/iscsi_sbps/HSM_groups_iscsi.md)
+
+If user/admin is not intended to limit the worker nodes for iSCSI SBPS i.e no `iscsi_worker` group is created, then all
+worker nodes will be configured as iSCSI targets. If the user does not want to configure any of the worker nodes as iSCSI targets, then the HSM group `iscsi_worker` can be kept empty which is not suggestible as no compute nodes can be booted.
+
+This group needs to be created at certain stages during CSM install/upgrade:
+
+During fresh install of CSM 1.7.0: `iscsi_worker` HSM group needs to be created before the `Configure management nodes with CFS` step of [CSM 1.7.0 installation](https://github.com/Cray-HPE/docs-csm/blob/release/1.7/install/configure_administrative_access.md#8-configure-management-nodes-with-cfs)
+
+During upgrade of CSM 1.7.0: `iscsi_worker` HSM group needs to be created before ‘management nodes rollout’ step during CSM upgrade using IUF. See [CSM 1.7.0 Upgrade](https://github.com/Cray-HPE/docs-csm/blob/ae9482ae85bc2eede3ac61079ffed5a00fb0a126/operations/iuf/workflows/management_rollout.md)
+
+Post install/upgrade if it is required to add any new worker node to the cluster or need to remove an existing worker node from the cluster, then see the [add/remove worker node](https://github.com/Cray-HPE/docs-csm/blob/release/1.7/operations/node_management/Add_Remove_Replace_NCNs/Add_Remove_Replace_NCNs.md#add-worker-storage-or-master-ncns).
+
+Post this, if it is required to add/remove any worker node to/from `iscsi_worker` group, then new node can be added or removed as below:
+
+Adding new node to `iscsi_worker` group:
+
+```bash
+cray hsm groups members create --id <xname of worker node> iscsi_worker
+```
+
+e.g
+
+```bash
+# cray hsm groups members create --id  x3000c0s18b0n0 iscsi_worker
+```
+
+e.g
+
+```text
+[[results]]
+URI = "/hsm/v2/groups/iscsi_worker/members/x3000c0s18b0n0"
+```
+
+Deleting the worker node from `iscsi_worker` group:
+
+```bash
+# cray hsm groups members delete <xname> iscsi_worker
+```
+
+e.g.
+
+```text
+# cray hsm groups members delete x3000c0s29b0n0 iscsi_worker
+
+ncn-m001:~ # cray hsm groups members delete x3000c0s29b0n0 iscsi_worker
+code = 0
+message = "deleted 1 entry"
+```
+
+Then re-run CFS on the worker nodes using below script:
```

Comment:
Once [this PR](https://github.com/Cray-HPE/docs-csm/pull/6126) merges, the script will be available.
The path to the script on the live system will be:
`/usr/share/doc/csm/scripts/operations/configuration/refresh_worker_iscsi_config.py`